static/app/components/replays/breadcrumbs/timelineGaps.tsx

```diff
@@ -3,66 +3,36 @@ import styled from '@emotion/styled';
 
 import {Tooltip} from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
-import {trackAnalytics} from 'sentry/utils/analytics';
 import toPercent from 'sentry/utils/number/toPercent';
-import {
-  getFrameOpOrCategory,
-  isBackgroundFrame,
-  isErrorFrame,
-  type ReplayFrame,
-} from 'sentry/utils/replays/types';
-import useOrganization from 'sentry/utils/useOrganization';
+import type {VideoEvent} from 'sentry/utils/replays/types';
 
 interface Props {
   durationMs: number;
-  frames: ReplayFrame[];
   startTimestampMs: number;
+  videoEvents: VideoEvent[];
 }
 
-// create gaps in the timeline by finding all columns between a background frame and foreground frame
-// or background frame to end of replay
-export default function TimelineGaps({durationMs, startTimestampMs, frames}: Props) {
-  const organization = useOrganization();
+export default function TimelineGaps({durationMs, startTimestampMs, videoEvents}: Props) {
   const ranges: Array<{left: string; width: string}> = [];
 
-  let start = -1;
-  let end = -1;
+  let start = startTimestampMs;
```

Comment:
Do we want to show if there's a gap in the beginning of the replay?

Reply:
i think if the replay starts in a state where there's no video we should show it. This can happen on the web if we get a metric or bit of data that happened outside of the pageload, so we do deal with it there too.

It's cool because we can say "some data happened at time=0, then at time=N we started to get the visuals"

Reply:
okok that makes sense! just worried that this might be confusing for customers